src/pipx/shared_libs.py

```diff
@@ -50,7 +50,8 @@ def create(self, verbose: bool = False) -> None:
 
             # ignore installed packages to ensure no unexpected patches from the OS vendor
             # are used
-            self.upgrade(pip_args=["--force-reinstall"], verbose=verbose)
+            pip_args = (pip_args or []) + ["--force-reinstall"]
```

Comment:
```suggestion
            pip_args = (pip_args or []).append("--force-reinstall")
```

Reply:
mm this won't work `append` works in-place. I could do
```py
pip_args = pip_args or []
pip_args.append("--force-reinstall")
```